vxquery-benchmark/src/main/resources/noaa-ghcn-daily/other_systems/basex_scripts/README

```diff
@@ -0,0 +1,9 @@
+BaseX local configuration options for the weather benchmark.
+The options attempt to match the processing and output of VXQuery.
+
+
+# Local Options
+CHOP = false
+DBPATH = /home/ecarman/disk1/basex/BaseXData
+REPOPATH = /home/ecarman/disk1/basex/BaseXRepo
+WEBPATH = /home/ecarman/disk1/basex/BaseXWeb
```

Comment:
I'm assuming that these overwrite during mvn package? Or is this README a sample guideline?